.github/workflows/documentation-update.yml

```
@@ -26,7 +26,7 @@ jobs:
         run: yarn run docgen
         working-directory: packages/contracts/
       - name: Format with prettier
-        run: yarn run prettier 'packages/contracts/docs/osx/03-reference-guide/**/*.md' --write
+        run: yarn run prettier 'packages/contracts/docs/develper-portal/03-reference-guide/**/*.md' --write
```

```suggestion
        run: yarn run prettier 'packages/contracts/docs/developer-portal/03-reference-guide/**/*.md' --write
```
Please fix the folder name and paths everywhere

---

done with af5827b4b0021a9d7320c9a9a4bdf7e5bdd71a88